mlir/test/IR/mlir-newline-after-attr.mlir

```diff
@@ -14,7 +14,8 @@
 // CHECK-NEXT: foo.vect_attr_1_start = dense<0> : vector<3xindex>,
```

Comment:
Can you add  --strict-whitespace and --match-full-lines to FileCheck to make this test a bit more useful?
https://llvm.org/docs/CommandGuide/FileCheck.html#cmdoption-FileCheck-strict-whitespace